src/lib/rest/HttpStatusCode.cpp

```diff
@@ -36,6 +36,7 @@ std::string httpStatusCodeString(HttpStatusCode code)
   case SccContextElementNotFound:            return "No context element found"; // Standard HTTP for 404: "Not Found"
   case SccBadVerb:                           return "Method Not Allowed";
   case SccNotAcceptable:                     return "Not Acceptable";
+  case SccConflict:                          return "TooManyResults";
```

Comment:
In issue #1199 I have solve this question including an if in OrionError.cpp, for example :

  else if (_reasonPhrase == "Length Required")
  {
    return "LengthRequired";
  }


And maintain White spaces in HttpStatusCode.cpp

  case SccLengthRequired:                    return "Length Required";

AS before has been mede with SccBadRequest